data.sql

```diff
@@ -1,7 +1,17 @@
+-- Previous INSERT statements
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
-
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8);
-
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
-
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
+
+-- Add new column species
+ALTER TABLE animals ADD COLUMN species CHAR(40);
+
+-- New INSERT statements
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species) VALUES ('Charmander', '2020-02-08', 0, false, -11, NULL);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species) VALUES ('Plantmon', '2021-11-15', 2, true, -5.7, NULL);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species) VALUES ('Squirtle', '1993-04-02', 3, false, -12.13, NULL);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species) VALUES ('Angemon', '2005-06-12', 1, true, -45, NULL);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species) VALUES ('Boarmon', '2005-06-07', 7, true, 20.4, NULL);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species) VALUES ('Blossom', '1998-10-13', 3, true, 17, NULL);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species) VALUES ('Ditto', '2022-05-14', 4, true, 22, NULL);
```

Comment:
👍 Great job with the data insertion! However, it's important to address the issue related to the missing `id` values in your queries:

- [ ] It appears that the `id` values are missing in the queries you've shared. Considering that your `animals` table does not currently have the functionality to generate `id` values automatically, you can use the `SERIAL PRIMARY KEY` constraint to automate this process. Currently, in the screenshot, I notice that you have an `id` value for all of them.To provide clarity and help reviewers understand how you obtained the `id` values in your screenshots, kindly update your SQL files with the queries you used for data insertion. Including this information will ensure a more thorough and accurate review of your project. Keep up the great work! 👍🔢📝
![image](https://github.com/CVILLA09/vet-clinic/assets/55840999/b875ef56-0747-412a-8219-95579154f480)